packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -613,36 +618,68 @@ class _FocusableActionDetectorState extends State<FocusableActionDetector> {
   void _handleMouseEnter(PointerEnterEvent event) {
     assert(widget.onShowHoverHighlight != null);
     if (!_hovering) {
-      // TODO(gspencergoog): remove scheduleMicrotask once MouseRegion event timing has changed.
-      scheduleMicrotask(() { setState(() { _hovering = true; _handleShowHoverHighlight(); }); });
+      _mayTriggerCallback(task: () {
+        setState(() { _hovering = true; });
```

Comment:
I'm not sure this needs to call `setState`, actually. It doesn't use `_hovering` in the `build` function.